.github/utils/pydoc-markdown.py

```diff
@@ -22,11 +22,10 @@ def load_search_paths():
             loader = config["loaders"][0]
             # `search_path` is a list but we always have only one item in Haystack
             search_path = loader["search_path"][0]
-            # we only need the relative path from the root, let's call `resolve` to
-            # get rid of the `../../` prefix
-            search_path = str(pathlib.Path(search_path).resolve())
-            # `resolve` will prepend a `/` to the path, remove it
-            paths[search_path[1:]] = fname
+            # we only need the relative path from the root, let's get rid of
+            # the `../../` prefix
+            search_path = search_path.replace("../", "")
+            paths[search_path] = fname
```

Comment:
Ok I think I need more context. `pathlib.Path` should be safer than a simple `replace("../", "")`. What's going on? If the paths are wrong then we should fix those instead of removing the `../`s from the path programmatically here.

Reply:
This used to work but now if you print the computed search_paths they make no sense. 
I am not sure what changed but while I was investigating the issue I realised we don't need to "resolve" the path, as the commit hook will always pass paths that are relative to the root of the repo, and the yaml files can't change without a deep refactoring of the tooling.
Considering that we control the yaml files, and considering that the markdown generation will be removed soon I thought replacing the `../` was the best approach.

Reply:
Makes sense! Thanks for the clarification